conf/repo/oca.yml

```diff
@@ -20,8 +20,9 @@ oca-custom:
     - "12.0"
     - "13.0"
     - "14.0"
+    - "18.0"
   category: Oca
-  default_branch: "14.0"
```

Comment:
I think this file shouldn't be modified

Reply:
fixed all the "main" branch cases (in the tool)

Reply:
should be fine now thanks to the key `manual_branch_mgmt: true`